lua/mason-lspconfig/init.lua

```diff
@@ -24,6 +24,10 @@ function M.setup(config)
         require "mason-lspconfig.ensure_installed"()
     end
 
+    require("mason-registry").register_package_aliases(_.map(function(server_name)
```

Comment:
For backwards compatibility we need to check that this function exists before using it:

```lua
if registry.register_package_aliases then
  -- ...
end
```